amqp/transport.py

```diff
@@ -54,7 +55,7 @@ class _AbstractTransport(object):
     """Common superclass for TCP and SSL transports"""
     connected = False
 
-    def __init__(self, host, connect_timeout):
+    def __init__(self, host, connect_timeout, write_timeout, read_timeout):
```

Comment:
@FlaPer87 now that i'm looking at this again, i remember that argument ordering for `SSLTransport` and `create_transport` will be a bit odd if we want to make everything backwards compatible internally. i've pushed a branch on my fork (https://github.com/dallasmarlow/py-amqp/pull/2) which has those changes, let me know if you would like to include those changes here.